testdata/scripts/imports.txt

```diff
@@ -1,11 +1,13 @@
 # Note that this is the only test with a module where we rely on the detection
 # of GOPRIVATE.
 
+env GOPRIVATE='test/*,rsc.io/*'
```

Comment:
see the note above; we don't want to set GOPRIVATE on purpose, to test the fallback

Reply:
if you do need this for the sake of the test, maybe we could move the "detect GOPRIVATE automatically" fallback test to a different file.

Reply:
Yeah maybe we should, or I could make a new test that tests import path obfuscation corner cases? What do you think?

Reply:
Up to you, I think either is reasonable. But if we're unsure, I'd go for the faster way, which is to reuse an existing test that has many modules/imports (or add them there).